My Series List/My Series List/Model/SeriesModel.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SeriesModel.swift
+//  My Series List
+//
+//  Created by Ivan Dias on 18/11/2021.
+//
+
+import Foundation
+
+struct Results: Codable {
+    let results: [Series]
+}
+
+struct Series: Codable {
+    let name: String?
+    let overview: String?
+    let poster_path: String?
```

Comment:
Can you use CodingKeys in order to use pascalCase instead of using underscores in variables names?